src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -75,7 +76,27 @@ public void start() {
 			appWindows.checkAndShowUpdateReminderWindow();
 		}
 
+		handleDokanySupportEndAndShowDialog();
+
 		launchEventHandler.startHandlingLaunchEvents();
 		autoUnlocker.tryUnlockForTimespan(2, TimeUnit.MINUTES);
 	}
+
+	private void handleDokanySupportEndAndShowDialog() {
+		var dokanyProviderId = "org.cryptomator.frontend.dokany.mount.DokanyMountProvider";
+		boolean dokanyFound = false;
+		if (settings.mountService.getValueSafe().contains(dokanyProviderId)) {
```

Comment:
You have to check for equality.

In this scenaria it there would be no bug (because Dokany is unique), still it is good practice because we want to ensure that only the settings with exactly that id is migrated.